limesurvey/extensions/filters.py

```diff
@@ -37,8 +38,16 @@ def run_filter(self, context, template_name):  # pylint: disable=unused-argument
             request, str(course.id), str(limesurvey_block.location),
             disable_staff_debug_info=True, course=course
         )
+
+        # Get display name and url for each LimeSurvey block
```

Comment:
This piece of code is pretty self-explanatory, I don't think we need the inline comment. We usually use them when we need to explain _why_ we're doing something, we shouldn't need to explain the what. Our implementation should speak for itself like in this case